.gitignore

```diff
@@ -1,7 +1,7 @@
 .frontmatter/
 
 # build output
-dist/
+_site/
```

Comment:
`_site` is the jekyll output. 
you can add `dist` which is the astro output but do not remove it.

`_site` is unnecessary for anyone who has a fresh checkout of the astro website.

Reply:
Oh, I see. I thought that was funny that you wouldn't have that already added. I'll remove that commit.